### DISCUSSION TITLE
Ignore this for a Live session - Education Purpose (2)

### DISCUSSION THREAD

**Discussion by codewithsadaf** [community/community#139524] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Ignore this for a Live session - Education Purpose (2)
Ignore this for a Live session - Education Purpose (2)

**Comment by umer4447** [comment]
Response #2

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/139525#discussioncomment-10730209) for reference to why this discussion was closed and locked. Thanks!
